csrc/scheduler/normalization_utils.cpp

```diff
@@ -1123,6 +1123,24 @@ bool compileTimeCheck(Fusion* fusion, ScheduleHeuristic schedule_heuristic) {
   return true;
 }
 
+int64_t getSharedMemoryOverheadPerBlock(
+    Fusion* fusion,
+    const std::vector<TensorView*>& reduction_tvs,
+    const int64_t max_threads_per_block) {
+  const auto& dev_prop = at::cuda::getCurrentDeviceProperties();
```

Comment:
Can you please give some more details about each of the overhead factor?

Reply:
added